Use server-side apply for swift-ring-files ConfigMap (#1382)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_adopting-the-object-storage-service.adoc

  object.ring.gz: $($CONTROLLER1_SSH "base64 -w0 /var/lib/config-data/puppet-generated/swift/etc/swift/object.ring.gz")
EOF
----
+
[NOTE]

> ====
> The `--server-side` flag is required because the {object_storage} ring files
> can be large enough to exceed the annotation size limit imposed by
> client-side `oc apply`.

Reviewer: ```suggestion
====
The `--server-side` flag is required if the {object_storage} ring files
are large enough to exceed the annotation size limit imposed by the
client-side `oc apply`.
```
Let me know if my edit changed the meaning. Could there be cases where the files do not exceed the size limit?

Author: Hi @klgill thanks for the review, I will apply the requested changes.
Yes, there might be cases, for small clusters, where this won't be a concern, but we don't know much about the environment we're going to adopt in advance, and this option can be applied both in the case of a small cluster and in case the cluster is big enough to exceed that limit.
Waiting for the command to fail and then suggest the option in case the command doesn't succeed is probably not the best user experience (I guess it's what we're trying to fix here), so I wanted to propose this flag and have a `Note` explaining the reason behind that.

Author: (Done)
